charts/federation-v2/charts/controllermanager/templates/deployment.yaml

```diff
@@ -22,20 +22,8 @@ spec:
     spec:
       containers:
       - args:
-{{- if .Values.featureGates }}
-        - --feature-gates={{ .Values.featureGates }}
-{{- end }}
 {{- if .Values.global.limitedScope }}
         - --limited-scope={{ .Values.global.limitedScope }}
-        - --federation-namespace=$(FEDERATION_NAMESPACE)
```

Comment:
Please restore this line and use it to configure the controller manager without condition.  The `--federation-namespace` arg should likely be retained to allow this.  

My understanding is that it is preferable to communicate values from the downward API via an arg rather than an environment variable.  Initial implementations of the controller manager used environment variables until @pmorie taught me the errors of my ways.